proposed/psr-8-hug/psr-8-hug.md

```diff
@@ -56,9 +64,21 @@ interface Huggable
 {
 
     /**
+     * Asks whether a hug will be reciprocated.
+     *
+     * Well-behaved Huggables MAY use this method to verify that their
+     * invocation of hug() will be handled by target object.
+     *
+     * @param Huggable $h
+     *   The object that desires a hug.
+     * @return boolean
+     */
+    public function willHug(Huggable $h);
```

Comment:
Good question — as can be seen from a few earlier commits, it started as `mayHug()` but that clearly implied the wrong direction of invocation.

`wouldhug()` Is probably the best option. According to the [English Grammar StackExchange](http://english.stackexchange.com/questions/3657/when-should-i-use-would-would-have-will-and-will-have), 'would' is a softer verb that implies future potential or conditional mood rather than commitment. In this case, that seems appropriate.